java/jdbc/statement-interceptor/interceptor/Readme.md

```diff
@@ -91,15 +91,15 @@ More than one actions can be defined (ex ['LOG','RAISE']). They are all executed
 ### Listener
 
 `com.oracle.jdbc.samples.interceptor.SQLStatementInterceptor` is a trace event listener responsible for analyzing SQL statements.  
-When a SQL statement is intercepted, the listener will loop through all defined rules and check if the SQL statement matches the rule. If a match is found (a risk is detected), the actions defined for that given rules are triggered.
+When an SQL statement is intercepted, the listener will loop through all defined rules and check if the SQL statement matches the rule. If a match is found (a risk is detected), the actions defined for that given rules are triggered.
 
 ### Configuration
 
-The configuration of rules and actions as a JSON file, the latter should contain an array of simple object with the following attributes:
+The configuration of rules and actions as a JSON file, the latter should contain an array of simple objects with the following attributes:
```

Comment:
Rules and actions are configured in a JSON file that contains an array of objects with the following attributes: